proto/connect/datastore_service/v1alpha1/datastore.proto

```diff
@@ -0,0 +1,50 @@
+// Taken from: https://github.com/spiffe/spire/blob/main/proto/spire/common/common.proto
```

Comment:
We've taken some protos from spire-api-sdk and put them in proto/spire/. I think this approach is ok if we want to keep this API purely for the datastore, but if we think we might open it up to other callers then it could go in proto/spire.

Reply:
yeah - I agonised a little about that :)